wordpress-mobile/WordPress-iOS
# Merges the latest from the Large Headers feature branch into develop.

Title has it.

What's important in this PR is to test the App with the feature flag `newNavigationBar` OFF, and make sure everything works fine.

Once this is merged, we can incrementally implement and push [what remains to be done](https://github.com/wordpress-mobile/WordPress-iOS/issues/15750) for `newNavigationBar` ON.

## Testing

Make sure the `newNavigationBar` feature flag is OFF by default.

- Make sure the UI in the My Site tab works fine.
- Make sure the Me button works fine.
- Make sure logging in and out between account with multiple sites, one site, and no sites works fine.

## PR submission checklist:
